osbs/build/build_request.py

```diff
@@ -27,6 +27,7 @@
 from osbs.constants import SECRETS_PATH, DEFAULT_OUTER_TEMPLATE, DEFAULT_INNER_TEMPLATE, DEFAULT_CUSTOMIZE_CONF
 from osbs.exceptions import OsbsException, OsbsValidationException
 from osbs.utils import looks_like_git_hash, git_repo_humanish_part_from_uri
+from osbs import __version__ as version
```

Comment:
This causes a landscape.io warning for remove_tag_and_push_registries() which takes a parameter of the same name.

Maybe we could import it as client_version?